app/assets/stylesheets/ama_layout/layout_components/banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner{
+	position: relative; /* This is needed for the positioning of the close icon on the blue banners*/
```

Comment:
If you copy the tab character that shows up here, you'll be able to paste it into your editor's search and should find them. My Textmate shows them as 2 spaces, so they look the same in there.